src/components/Organisms/TodoCard/index.jsx

```diff
@@ -42,6 +38,15 @@ const TodoCard = () => {
     }
   };
 
+  useEffect(() => {
+    const data = JSON.parse(localStorage.getItem("taskData") ?? []);
```

Comment:
`SyntaxError: Unexpected token o in JSON at position 1` は、現状の実装に起因するものだと思います。

![image](https://github.com/OzawaSachio/sachi-frontend-basic-2022-todoapp/assets/83479175/2ec5b1c5-bf03-4f32-b8ac-86c3c8bed541)
画像に示したように、 `[]` を `JSON.parse()` に入れると unexpected Json Input のエラーが出てしまいます。

初回の表示の際は `taskData` が localStrageに存在しないため `JSON.parse([])` となり、上記のエラーが出てしまうわけです。

```suggestion
    const data = JSON.parse(localStorage.getItem("taskData")) ?? [];
```

提案したように修正して、シークレットウインドウで開いてみてください。
おそらくエラーが出なくなると思います。

（二回目以降は46行目の記述によりオブジェクトが入るため、エラーが出なくなります。）

Reply:
Chromatic が エラーになっているのも、この修正を入れれば直るかと！